### PR TITLE
Fixed groupme icon

### LINKED ISSUE #7: Fix the GroupMe Icon
The GroupMe icon is, for some reason, not showing up on the page. Unlike the Discord and Email icons, the GroupMe icon is not using FontAwesome.

Acceptance Criteria
1) Locate a new/improved GroupMe SVG to replace the GroupMe PNG
2) Modify `LabeledButton` to make it able to display SVGs that aren't FontAwesome, without breaking any of the other `LabeledButton` components
3) Make sure it appears on the page correctly, with the correct size and color

### PR DESCRIPTION
**Pull request by harmless-tech**
Issue #7

### CODE REVIEW

File: src/components/lib/Label.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ export interface IconProps {
      */
     fillStyle?: "solid" | "regular" | "light" | "duotone" | "brand";
     icon: string;
+    fa?: "true" | "false";
```
Comment (DonIsaac) [inline src/components/lib/Label.tsx]: make this a boolean

File: src/components/lib/Label.tsx [hunk 2]
```diff
@@ -46,12 +47,12 @@ export const IconLabel: FC<IconLabelProps> = ({
...
-    <span
-        className={classNames([
-            "fa" + fillStyle.charAt(0),
-            icon.startsWith("fa-")
-                ? icon
-                : "fa-" + icon
-        ])}
-    />;
+export const Icon: FC<IconProps> = ({ fillStyle = "", icon = "", fa = "true" }) => {
+    if(fa === "false") {
```
Comment (DonIsaac) [inline src/components/lib/Label.tsx]: Make this `if (!fa)` when you make `fa` a boolean

**Review by DonIsaac — CHANGES_REQUESTED** [review]

**Comment by lovendatj** [comment]
It looks goochi

**Comment by lovendatj** [comment]
Opening pull request for merg into dev

File: src/components/lib/Label.tsx [hunk 3]
```diff
@@ -11,6 +11,7 @@ export interface IconProps {
      */
     fillStyle?: "solid" | "regular" | "light" | "duotone" | "brand";
     icon: string;
+    fa?: true | false;
```
Comment (DonIsaac) [inline src/components/lib/Label.tsx:14]: this should be `boolean`

Comment (harmless-tech) [reply src/components/lib/Label.tsx:14]: I fixed this in my fork, but I think the pull request needs to be reopened for the new commit to show up in the pull request.

Comment (DonIsaac) [reply src/components/lib/Label.tsx:14]: Please re-open the PR then

**Pull request closed without merging**